chia/server/server.py

```diff
@@ -493,29 +493,54 @@ async def start_client(
             connection_type_str = ""
             if connection.connection_type is not None:
                 connection_type_str = connection.connection_type.name.lower()
-            self.log.info(f"Connected with {connection_type_str} {target_node}")
-            if is_feeler:
+            if not is_feeler:
+                self.log.info(f"Connected with {connection_type_str} {target_node}")
+            else:
+                self.log.debug(f"Successful feeler connection with {connection_type_str} {target_node}")
                 asyncio.create_task(connection.close())
             return True
         except client_exceptions.ClientConnectorError as e:
-            self.log.info(f"{e}")
+            if not is_feeler:
+                self.log.info(f"{e}")
+            else:
+                self.log.debug(f"Feeler connection error. {e}")
         except ProtocolError as e:
             if connection is not None:
                 await connection.close(self.invalid_protocol_ban_seconds, WSCloseCode.PROTOCOL_ERROR, e.code)
             if e.code == Err.INVALID_HANDSHAKE:
-                self.log.warning(f"Invalid handshake with peer {target_node}. Maybe the peer is running old software.")
+                if is_feeler:
+                    self.log.warning(
+                        f"Invalid handshake with peer {target_node} during feeler connection. "
+                        f"Maybe the peer is running old software."
+                    )
+                else:
+                    self.log.warning(
+                        f"Invalid handshake with peer {target_node}. Maybe the peer is running old software."
+                    )
             elif e.code == Err.INCOMPATIBLE_NETWORK_ID:
-                self.log.warning("Incompatible network ID. Maybe the peer is on another network")
+                if is_feeler:
+                    self.log.warning(
+                        "Incompatible network ID during feeler connection. Maybe the peer is on another network"
+                    )
+                else:
+                    self.log.warning("Incompatible network ID. Maybe the peer is on another network")
             elif e.code == Err.SELF_CONNECTION:
                 pass
             else:
-                error_stack = traceback.format_exc()
-                self.log.error(f"Exception {e}, exception Stack: {error_stack}")
+                if is_feeler:
+                    error_stack = traceback.format_exc()
+                    self.log.error(f"Feeler connection exception {e}, exception Stack: {error_stack}")
+                else:
+                    error_stack = traceback.format_exc()
+                    self.log.error(f"Exception {e}, exception Stack: {error_stack}")
```

Comment:
This could be written as a single line like

self.log.error(f"{'Feeler connection' if is_feeler else ''} Exception {e}, exception Stack: {error_stack}")

Reply:
Similar things can be done for the earlier strings as well - I'm not sure logging whether it is a feeler or not is that important but I think a one liner with a ternary might be a little better...

Reply:
A simpler way might be just 

`self.log.xxx(f"Same string : feeler {is_feeler}")`